src/sentry/integrations/github/client.py

```diff
@@ -55,8 +57,43 @@ def __repr__(self) -> str:
         return f"GithubRateLimitInfo(limit={self.limit},rem={self.remaining},reset={self.reset})"
 
 
+class GithubSetupApiClient(IntegrationProxyClient):
+    """
+    API Client that doesn't require an installation.
+    This client is used during integration setup to fetch data
+    needed to build installation metadata
+    """
+
+    base_url = "https://api.github.com"
+    integration_name = "github_setup"
+
+    def __init__(self, access_token: str | None = None, verify_ssl: bool = True):
+        super().__init__(verify_ssl=verify_ssl)
+        self.jwt = get_jwt()
+        self.access_token = access_token
+
+    @control_silo_function
+    def authorize_request(self, prepared_request: PreparedRequest) -> PreparedRequest:
+        if any(url in prepared_request.path_url for url in JWT_AUTH_ROUTES):
+            prepared_request.headers["Authorization"] = f"Bearer {self.jwt}"
+        else:
+            prepared_request.headers["Authorization"] = f"token {self.access_token}"
```

Comment:
I honestly don't know if there's a difference between saying `Bearer` vs `token` in the header

Reply:
for a while (idk if it's still true) github completely ignored the first word there so you could put anything

Reply:
did some manual testing with just using `Bearer <token>` and there seemed like no diff, so should be good. Tbf the [`GitHubProxyClient` ](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/client.py#L172)only uses `Bearer` too 🤷‍♀️